src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="ArrayDepthTooLarge" xml:space="preserve">
-    <value>CurrentDepth ({0}) is larger than the maximum configured depth of {1}. Cannot read next JSON array.</value>
+    <value>Depth has exceeded the maximum configured depth of {0}. Cannot read next JSON array.</value>
```

Comment:
"The maximum configured depth of {0} has been exceeded. Cannot read the next JSON arrray."  seems more natural.